### DISCUSSION TITLE
Suggestion: Smousss Factorize, to generate Model Factories using artificial intelligence.

### DISCUSSION THREAD

**Discussion by benjamincrozat** [HassanZahirnia/laravel-package-ocean#67] [Package Suggestions]
I recently saw this site on Twitter and I absolutely love the UI. Great job there! Naturally, I want to contribute with my own packages. The first one is [Smousss Factorize](https://github.com/smousss/laravel-factorize).

One of the pain point I have when I'm doing freelance work on existing projects is that factories are often missing because previous developers weren't skilled enough with Laravel. Missing factories prevent you writing seeders and more importantly, automated tests.

Generating factories without AI is possible, but artificial intelligence brings the missing human-like touch. And it's even better when the AI has access to your file system.

I think it'd be interesting to start a new section for artificial intelligence on the site, because it brings so much to the table. 👌

Learn more about [Smousss Factorize](https://github.com/smousss/laravel-factorize).

**Comment by HassanZahirnia** [comment]
Hey Benjamin! 😊👋
Thanks for your kind words! 💙
I agree with you that supercharging development with AI is definitely worth it and I'm glad you took steps by creating this package 🙌
I've added your package to the database, even though I don't usually add packages that are super new, I made an exception here since I want to move the community with the usage of AI.

As for adding a new category for AI, I'm not against the idea, but at the moment I want to keep the UI clean and simple and not clutter the category sidebar with dozens and dozens of different categories. In the future I might add such category, if there were more AI powered tools.
